### PR TITLE
CommVault Implementation Guide

### PR DESCRIPTION
**Pull request by kashifzeeshan**
Updated the Implementation Guide as per the Review

## What Changed?

Please list, at a high level, what changed in your branch. If you changed content, include the product, section, and version as applicable. **Please remove the user instructions including the examples before merging the PR.**

**Examples**

- Fixed typo in EPAS 13 epas_inst_linux guide
- Added more detail to ARK 3.5 getting started guide introduction
- Fixed broken link on `/supported-open-source/pgbackrest/05-retention_policy/` page

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+8. Creating a Server Backup Plan in Core Setup: In the server backup plan, define where data is stored and how often to back up data.
+9. In the Plan name box,type the name of the plan e.g EPAS Server Plan.
+10. In the Backup destinations section, select the Storage and Retention Period.
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup PostgreSQL Database Server
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:43]: This should be Setup EDB Postgres Advanced Server

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+1. From the navigation pane, go to Protect > Databases.
+2. Click Add server.
+3. Select the database type which in this case is PostgreSQL.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:46]: How about 
Select the database type for EDB Postgres Advanced server, which in this case is PostgreSQL.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd1.png">
+</p>
+
+4. In the Database server name box, enter the server name.
+5. In the Username and Password boxes, enter the credentials to connect to the server.
+6. From the Plan list, select the server plan to use for the database server.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:54]: select the server plan created in step 8 under Core Server Wizard section. for use with EDB Postgres Advanced Server.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd2.png">
+</p>
+
+7. Once server is added, then a Job will run to install client on the Server. Below screenshots display the entire process were a Job is created to install Client on the Server and then Dowloaded it and installed it.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:60]: Reword step 7
Once the server is added, a "Job" will run to install a client on the database server.  The screen shots below display the process from Job creation to software installation.

or something like this.

For each screen shot can you add a label underneeth it that tells the reader what the screen shot it. as example.
job creation
installation package created
Download the software
install the software.
Note- I can't see the images clear enough to know what they are, so I made up the examples.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+1. From the navigation pane, go to Protect > Databases > DB Instances.
+2. Click Add instance, and then select PostgreSQL.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/CreateInstance1.png">
+</p>
+
+3. From the Server name list, select a server where you want to create the new instance.
+4. In the Instance Name box, type the EDB Postgres Advanced Server instance name.
+5. From the Plan list, select a server plan.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:88]: Should this be?

Select the server plan created in step 8 under Core Server Wizard section. for use with EDB Postgres Advanced Server.

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 6]
```diff
@@ -0,0 +1,188 @@
+---
+title: 'Using Commvault Backup & Recovery'
+description: 'Walkthroughs of multiple Commvault Backup & Recovery usage scenarios'
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
+
+## Using Commvault Backup & Recovery
+
+Commvault provides the two methods of taking the backup from EDB Postgres Advanced Server and restoring it.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:10]: Commvault provides the two methods 
chang
Commvault provides two methods ....       Don't need "the'

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+1. DumpBasedBackupSet Backup and Restore
+2. FSBasedBackupSet Backup and Restore
+
+Following examples shows how to use both methods of taking and restoring the backups.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:15]: remove this while line.   Don't need it.

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+### DumpBasedBackupSet Backup and Restore
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:17]: Would be good to have this 
"DumpBasedBackupSet Backup and Restore"
and
"FSBasedBackupSet Backup and Restore"
as links on the right hand side so people could click them and go right to those section.  Ask Jasmer about this if you don't know how to enable it.

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+4. Select a date from the calendar, and then click Restore.
+5. Backup Content Screen will display the databases to be restored, select the required database to restore or select all of them to restore all.
+
+<p align="center">
+  <img  alt="Backup Content Screen" src="Images/Dumprestore2.png">
+</p>
+
+6. Click Restore.
+7. From Restore Options Screen, select the Destination Server and Destination Instance and click on submit.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:86]: From the Restore ....

add "the"

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+
+<p align="center">
+  <img  alt="Restore Job" src="Images/Dumprestore4.png">
+</p>
+
+<p align="center">
+  <img  alt="Restore Job" src="Images/Dumprestore5.png">
+</p>
+
+9. Once the Restore is completed successfully, login to the EDB Postgres Advanced Server Instance and check the restore operation recovered the data.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:102]: check that the restore operation ...

add "that"

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:103]: What is this screen showing?  I don't think it helps as it appears to be just an after restore screen shot.  I am not sure it is of much value with out a before.

If we showed the database with say two records (Before restore). then show database with 4 records (after restore) then it has value as we now see the restore worked.   Or you could show you have one database then restore and you now have two databases.  something like that.

File: advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+4. Click Local on the radio button for Type.
+5. In the Backup location box, click Browse to assign a path where backups will be stored.
+6. To enable deduplication on the storage, move the Use deduplication toggle key to  the right, and in the Deduplication DB location box, browse to select the path to  the deduplication database.
+7. To move to the next setup option, click Save
+
+<p align="center">
+  <img  alt="Setup Storage Pool/Disk Storage" src="Images/DiskStorageConf.png">
+</p>
+
+8. Creating a Server Backup Plan in Core Setup: In the server backup plan, define where data is stored and how often to back up data.
```
Comment (sethijs) [inline advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx:34]: I would rewrite step #8 this way: 

"In order to create a server backup plan in Core Setup, define where the data is stored and how often must the data undergo a backup."

File: advocacy_docs/partner_docs/CommVaultGuide/03-SolutionSummary.mdx [hunk 13]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Commvault enables businesses to proactively simplify and manage the complexity of their continuously evolving and growing data environments, whether your data is on-premises or in the cloud.
```
Comment (sethijs) [inline advocacy_docs/partner_docs/CommVaultGuide/03-SolutionSummary.mdx:5]: This sentence is a bit of a run-on. I would simplify it like this: 

"Commvault enables your business to streamline management of its continuously evolving data environment, whether the data is on-premises or in the cloud."

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 14]
```diff
@@ -0,0 +1,187 @@
+---
+title: 'Using Commvault Backup & Recovery'
+description: 'Walkthroughs of multiple Commvault Backup & Recovery usage scenarios'
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
```
Comment (sethijs) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:6]: Remove this line

File: advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx [hunk 15]
```diff
@@ -0,0 +1,187 @@
...
+
+## Using Commvault Backup & Recovery
```
Comment (sethijs) [inline advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx:8]: Remove this header, we already have it as the page title

**Pull request closed without merging**